<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCommConfigUdpClientDlg</class>
 <widget class="QDialog" name="QCommConfigUdpClientDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>本机IP地址</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="m_editRemotePort"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>组播IP地址</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="m_editMulticastIP"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_editLocalPort"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_editLocalIP"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>本机端口号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>远程端口号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>远程IP地址</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_editRemoteIP"/>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="m_chkIsBroadcast">
       <property name="text">
        <string>是否使用广播</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
